NeoMathEngine/src/CPU/x86/CpuX86MathEngineVectorMathPrivate.h

```diff
@@ -1174,6 +1174,45 @@ inline void vectorHSwish( const float* first, float* result, int vectorSize )
 	}
 }
 
+//------------------------------------------------------------------------------------------------------------
+
+inline void vectorHardSigmoid( const float* first, float* result, float slope, float bias, int vectorSize )
+{
+	int sseSize;
+	int nonSseSize;
+	checkSse( vectorSize, sseSize, nonSseSize );
```

Comment:
I don't understand your passion about `chekSse()` ang `GetCount4()`. Look at #858

Reply:
It's not a passion, it's laziness)
This code was moved from the other file as-is.

Reply:
Btw, fixed

Reply:
What about `GetCount4()`?